ledger/sandbox/src/main/scala/com/digitalasset/platform/logging/LoggingContext.scala

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.logging
+
+import net.logstash.logback.argument.StructuredArgument
+import net.logstash.logback.marker.MapEntriesAppendingMarker
+import org.slf4j.Marker
+
+import scala.collection.JavaConverters.mapAsJavaMapConverter
+
+object LoggingContext {
+
+  def newLoggingContext[A](f: LoggingContext => A): A =
+    f(new LoggingContext(Map.empty))
+
+  def newLoggingContext[A](kv: (String, String), kvs: (String, String)*)(
+      f: LoggingContext => A): A =
+    f(new LoggingContext(Map(kv +: kvs: _*)))
```

Comment:
I find `x.toMap` more readable than `Map(x: _*)`.

```suggestion
    f(new LoggingContext((kv +: kvs).toMap))
```